pkg/resources/opnicluster/elastic/indices/opni_settings.go

```diff
@@ -17,6 +17,10 @@ const (
 	drainStatusIndexPrefix       = "opni-drain-model-status-v0.1.3"
 	drainStatusIndexAlias        = "opni-drain-model-status"
 	drainStatusIndexTemplateName = "opni-drain-model-status_rollover_mapping"
+	metricPolicyName             = "opni-metric-policy"
+	metricIndexPrefix            = "opni-metric-v0.1.3"
```

Comment:
We should probably set this to `opni-metric-v0.3.0` to match the release

Reply:
add a const releaseVersion=v0.3.0 to match our next release.